FitGoal/FitGoal/Singup&Singin/CustomUIViews/AuthenticationLinkStack.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  LoginLinkStack.swift
+//  FitGoal
+//
+//  Created by Eliany Barrera on 20/3/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+protocol AuthenticationTypeDelegate: AnyObject {
+    func userDidSelectAuthenticationType()
+}
+
+class AuthenticationLinkStack: UIStackView {
```

Comment:
Same as before, probably you don't need to expose that you are using a stack view behind the scenes, so maybe `AuthenticationLinkView` is a better name. Other question, usually you have a class and its delegate has a similar name. However, in this case you aren't doing it that way. Why?

Reply:
No reason, I'm still getting used to naming things and sometimes, after I write a name and some logic for the object, I realise that It's better to name it in another way So, sometimes I rename parts of the logic, but let others behind. ;-)